bitcoinperf.yaml

```diff
@@ -0,0 +1,62 @@
+---
+
+# Where various data output is dumped
+artifact_dir: /tmp/output
+
+# Cache a built bitcoin src directory and restore it from the cache on
+# subsequent runs.
+cache_build: false
+
+# Set to true to disable cache dropping and various safety checks.
+no_caution: false
```

Comment:
nit: instead of double negative, it might be easier to parse without any negative

e.g.
```yml
saftety_checks: true

Reply:
yup, good idea